lib/rogue/vision_helper.rb

```diff
@@ -1,6 +1,6 @@
 module Rogue
   class VisionHelper
-    def initialize(map)
+    def initialize(map=Rogue.map)
```

Comment:
Looking back at this now, I think this is not a good idea.. Will refactor it so that a map is passed to the helper whenever it needs to be used.